apps/admin/frontend/src/screens/manual_data_entry_screen.tsx

```diff
@@ -721,6 +721,7 @@ export function ManualDataEntryScreen(): JSX.Element {
         );
       })}
       <P weight="semiBold">
+        <Icons.Warning />{' '}
```

Comment:
It looks like the logic has changed here to always show the warning icon - what's the thinking?

Reply:
That was a mistake, thanks for catching. Fixed: f6fdd87e3